### PR TITLE
cni networking plugin now support portmapping and bandwidth

### PR DESCRIPTION
**Pull request by Lion-Wei**
Document that cni networking plugin now support portmapping and pod bandwidth control.

xref: https://github.com/kubernetes/kubernetes/pull/63194

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -45,7 +45,48 @@ If there are multiple CNI configuration files in the directory, the first one in
...
+}
+```
+
+
+##### Support bandwidth contol
+
+CNI networking plugin now can support pod ingress and egress bandwidth control, you can use the officical [bandwidth](https://github.com/containernetworking/plugins/tree/master/plugins/meta/bandwidth)
+plugin offered by CNI plugin team, or use your own plugin with bandwidth contol function.
+
+If you want enable support bandwidth contol, the `bandwidth capability` need to be specified in your `cni-conf-dir`.
```
[the remark this answers is not in the document]
Comment (m1093782566) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: I would be better to tell the default location of `cni-conf-dir`?

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 2]
```diff
@@ -45,7 +45,48 @@ If there are multiple CNI configuration files in the directory, the first one in
...
+For example:
+
```
Comment (m1093782566) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: Can we add a Pod example with ingress and egress bandwidth limitation?

**Comment by m1093782566** [comment]
cc @buptliuwei

**Comment by Lion-Wei** [comment]
@m1093782566 Thanks, all comment are addressed. : )

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 3]
```diff
@@ -45,7 +45,90 @@ If there are multiple CNI configuration files in the directory, the first one in
...
+      },
+      "policy": {
+        "type": "k8s"
+      },
+      "kubernetes": {
+        "kubeconfig": "/etc/cni/net.d/calico-kubeconfig"
+      }
+    },
+    {
+      "type": "portmap",
```
Comment (buptliuwei) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: should be ?
  "type": "bandwidth",
  "capabilities": {"trafficShaping": true}

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 4]
```diff
@@ -45,7 +45,90 @@ If there are multiple CNI configuration files in the directory, the first one in
...
+        "subnet": "usePodCidr"
+      },
+      "policy": {
+        "type": "k8s"
+      },
+      "kubernetes": {
+        "kubeconfig": "/etc/cni/net.d/calico-kubeconfig"
+      }
+    },
+    {
```
Comment (buptliuwei) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: should be?
"type": "bandwidth",
"capabilities": {"trafficShaping": true}

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 5]
```diff
@@ -45,7 +45,90 @@ If there are multiple CNI configuration files in the directory, the first one in
...
+      "type": "portmap",
+      "capabilities": {"portMappings": true}
```
Comment (buptliuwei) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: should be
"type": "bandwidth",
"capabilities": {"trafficShaping": true}
?

Comment (m1093782566) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: This example is about portmap plugin :)

Comment (Lion-Wei) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: I noticed this part is example for portmap. Though I do mixed in traffic shaping part. : )

Comment (Lion-Wei) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: I noticed this example is for port mapping. Though I do mixed up in traffic shaping part. :smile:

**Comment by m1093782566** [comment]
LGTM from technical view.

/lgtm

ping @zacharysarah @kbarnard10 for approval.

**Comment by Lion-Wei** [comment]
/assign @chenopis 
PTAL, thanks.

**Comment by Lion-Wei** [comment]
@chenopis Thanks.

**Pull request merged**